internal/sbomsync/builder.go

```diff
@@ -78,6 +80,38 @@ func (b sbomBuilder) AddPackages(p ...pkg.Package) {
 	b.onWriteEvent()
 }
 
+func (b sbomBuilder) DeletePackages(ids ...artifact.ID) {
+	b.lock.Lock()
+	defer b.lock.Unlock()
+
+	toDelete := make(map[artifact.ID]struct{})
```

Comment:
nit/rename: I was looking at the end of the function based on the tense and realized it was done at the top already
```suggestion
	deleted := make(map[artifact.ID]struct{})
```